indigo_app/views/works.py

```diff
@@ -638,6 +638,8 @@ def get_form(self, form_class=None):
     def get_context_data(self, **kwargs):
         context = super(BatchAddWorkView, self).get_context_data(**kwargs)
         context['bulk_creator'] = self.bulk_creator
+        if self.bulk_creator.is_gsheets_enabled:
```

Comment:
This isn't necessary, the template can just use `bulk_creator.share_with` since the bulk creator is already in the context.